configs/foveabox/fovea_ms_r101_fpn_4gpu_2x_align.py

```diff
@@ -0,0 +1,135 @@
+# model settings
+model = dict(
+    type='FOVEA',
+    pretrained='torchvision://resnet101',
+    backbone=dict(
+        type='ResNet',
+        depth=101,
+        num_stages=4,
+        out_indices=(0, 1, 2, 3),
+        frozen_stages=1,
+        style='pytorch'),
+    neck=dict(
+        type='FPN',
+        in_channels=[256, 512, 1024, 2048],
+        out_channels=256,
+        start_level=1,
+        num_outs=5,
+        add_extra_convs=True),
+    bbox_head=dict(
+        type='FoveaHead',
+        num_classes=81,
+        in_channels=256,
+        stacked_convs=4,
+        feat_channels=256,
+        strides=[8, 16, 32, 64, 128],
+        base_edge_list=[16, 32, 64, 128, 256],
+        scale_ranges=((1, 64), (32, 128), (64, 256),
+                      (128, 512), (256, 2048)),
+        sigma=0.4,
+        with_deform=True,
+        norm_cfg=dict(type='GN',
+                      num_groups=32,
+                      requires_grad=True),
```

Comment:
Fovea head uses `GN` by default. The name of the config should clarify it.

Reply:
> Fovea head uses `GN` by default. The name of the config should clarify it.

fixed